dm_regional_app/templates/dm_regional_app/views/costs.html

```diff
@@ -243,6 +244,9 @@ <h2 class="accordion-header" id="headingThree">
                         </table>
                     </div>
                     <div class="d-flex bd-highlight mb-3">
+                      {% if proportions.columns|length == 4 %}
+                      <div class="p-2 bd-highlight"><a class="btn btn-warning" href="clear_proportions?next_url_name=costs">Clear all proportion adjustments</a></div>
```

Comment:
A nitpick, but I think putting the <a> tags on a newline could improve readability
```suggestion
                      <div class="p-2 bd-highlight">
                          <a class="btn btn-warning" href="clear_proportions?next_url_name=costs">Clear all proportion adjustments</a>
                      </div>
```